src/components/views/ViewsListTable.tsx

```diff
@@ -16,10 +17,35 @@ const ViewsListTable: React.FunctionComponent = () => {
     const router = useRouter();
     const [selectedViewToDelete, setSelectedViewToDelete] = useState<number | undefined>(undefined);
     const { orgId } = router.query;
+    const { showConfirmDialog } = useContext(ConfirmDialogContext);
     const { showSnackbar } = useContext(SnackbarContext);
     const deleteMutation = viewsResource(orgId as string).useDelete();
     const viewsQuery = viewsResource(orgId as string).useQuery();
 
+    useEffect(() => {
+        if (selectedViewToDelete) deleteView();
```

Comment:
I'm not a fan of single-line conditionals (and can go on at great length as to why, but I'll spare you that unless you feel strongly about it and want to hear my motivations 😊). I think this should be reformatted as:

```typescript
if (selectedViewToDelete) {
    deleteView();
}
```

Reply:
done